package-lock.json

```diff
@@ -592,8 +722,7 @@
     },
     "node_modules/@babel/core/node_modules/json5": {
       "version": "2.2.2",
-      "resolved": "https://registry.npmjs.org/json5/-/json5-2.2.2.tgz",
-      "integrity": "sha512-46Tk9JiOL2z7ytNQWFLpj99RZkVgeHf87yGQKsIkaPz1qSH9UczKH1rO7K3wgRselo0tYMUNfecYpm/p1vC7tQ==",
+      "license": "MIT",
```

Comment:
@perunt Could you help to review package-lock.json file? There are many weird changes

Reply:
Hmm, it seems that npm install removes the resolved and integrity properties from package-lock.json when installing from the cache. I've reverted the changes—thanks for pointing that out!